data/mods/Aftershock/items/gunmods/laser_gunmods.json

```
@@ -0,0 +1,29 @@
+[
+  {
+    "id": "electrolaser_conversion",
+    "type": "GUNMOD",
+    "copy-from": "electrolaser_conversion",
+    "name": { "str": "electrolaser conversion" },
+    "description": "A set of high-tech electronics and optics.  These converts a laser pistol into a less-lethal electrolaser capable of stunning targets, at the cost of a decreased damage output and increased power consumption.",
```

```suggestion
    "description": "A set of high-tech electronics and optics.  These convert a laser pistol into a less-lethal electrolaser capable of stunning targets, at the cost of decreased damage output and increased power consumption.",
```
Also, where does the stunning come in?

---

The stunning properties are defined on the main definition in vanilla's json, this one is really just handling compatibility.

---

How do I suggest a change on a suggested change? `This converts a laser pistol...` is grammatically correct, as the singular article 'this' refers to the singular noun 'set of X.'